package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "pretest": "standard",
     "test": "jest",
-    "start": "webpack-dev-server --mode development",
-    "build": "webpack --mode production"
+    "start": "NODE_ENV=development webpack-dev-server --mode development",
+    "build": "NODE_ENV=production webpack --mode production"
```

Comment:
don't the `mode` flags here duplicate the work of setting NODE_ENV?

Reply:
I wanted to use the env variables to change the path of the file to be used in the `index. html` file [generated](https://github.com/piyushagwl007/ah-swagger-plugin/blob/6e2df884def8bab71efe53d7b0fa4698ce09bb6d/webpack.config.js#L41-L44).

Using `--mode development` or `--mode production` I was unable to do that.
Should I remove these flags or look for some different approach ? like creating different config files for production and development?
```
const devMode = process.env.NODE_ENV !== 'production'
...

  output: {
    path: path.resolve(__dirname, 'public/swagger'),
    publicPath: devMode ? '../../public/swagger' : '/swagger/',
    filename: '[name]-bundle.min.js'
}
```

Reply:
That makes sense!  Leave it!